spread.yaml

```diff
@@ -9,7 +9,7 @@ environment:
     SNAP_REEXEC: 0
     SPREAD_STORE_USER: "$(HOST: echo $SPREAD_STORE_USER)"
     SPREAD_STORE_PASSWORD: "$(HOST: echo $SPREAD_STORE_PASSWORD)"
-    LANG: "$(echo $LANG)"
+    LANG: "$(echo ${LANG:-C.UTF-8})"
```

Comment:
This isn't right, we will still run with `LANG` unset but happily print `C.UTF-8` here as if it was. I think you want to use ${LANG:=C.UTF-8}` instead which will assign it.